air_quality_monitoring/raqmn/raqmn_data/templates/raqmn_data/plot.html

```diff
@@ -35,8 +35,8 @@
 	{% endfor %}
 	var graph = new Rickshaw.Graph({
```

Comment:
I understood that this is what it was doing, but I don't see a lot of discussion/documentation on this option. Can you add some comments on the architecture here and how you are querying the timeseries database?